src/components/BottomNav/BottomNav.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import * as S from './BottomNav.style';
+import { IcDeliver, IcLocation } from '@assets/svgs';
+
+const BottomNav = () => {
+  const [isOpen, setIsOpen] = useState(true);
+  const [lastScrollY, setLastScrollY] = useState(0);
+
+  // 스크롤 방향 감지 함수
+  const handleScroll = () => {
+    const currentScrollY = window.scrollY;
+    currentScrollY > lastScrollY ? setIsOpen(true) : setIsOpen(false);
```

Comment:
삼항 연산자는 조건에 따라서 반환하는 값을 정해주는 역할을 하는데요!
`setState`는 반환값이 없습니다!!
따라서 if문을 사용하는 것이 더 알맞은 문법입니다!

코드가 길어지는게 싫다면 참일 때 참으로, 거짓일 때 거짓으로 set을 주고 있으니까 아래처럼 작성해도 좋을 것 같습니다 :)
```suggestion
    setIsOpen(currentScrollY>lastScrollY);
```

Reply:
헉 그렇군요! 좋은 피드백 감사합니다👏